deckhouse-controller/pkg/debug/debug.go

```diff
@@ -76,7 +76,7 @@ func createTarball() *bytes.Buffer {
 		{
 			File: "global-values.json",
 			Cmd:  "bash",
-			Args: []string{"-c", `deckhouse-controller global values -o json | jq .`},
+			Args: []string{"-c", `deckhouse-controller global values -o json | jq '.internal.modules.kubeRBACProxyCA = "REDACTED" | .modulesImages.registry.dockercfg = "REDACTED"'`},
```

Comment:
NIT: I would like to see some extra field in the `Command` struct, like `FilterFn` or `FilterFields` or `JSONFilter` which will be called in the `cmd.Save` function.
Then you can realize the filter in a few ways - sjson, jq, manual filtering, whatever

Reply:
Like:
```golang
{
			File: "global-values.json",
			Cmd:  "bash",
			Args: []string{"-c", `deckhouse-controller global values -o json`}
            FilterFunc: func(s string) string {
              val, _ := sjson.Set(s, "internal.modules.kubeRBACProxyCA", "REDACTED")
              return val
            },
```